Sources/SwiftKuery/Column.swift

```diff
@@ -84,8 +114,23 @@ public class Column: Field, IndexColumn {
         self.collate = collate
     }
     
-    /// Build the column using `QueryBuilder`.
-    ///
+    //MARK: Column Decription Functions
+    
+    /**
+     Function to build a String representation for referencing a `Column` instance.
+     A `QueryBuilder` is used handle variances between the various database engines and produce a correct SQL description.
+     This function is required to obey the `Buildable` protocol.
+     ### Usage Example: ###
+     In this example, `QueryBuilder` and a `PersonTable` (as defined at the top of this class) instances are initialized.
+     The build function is then called to produce a String description and the results are printed.
+     ```swift
+     let queryBuilder = QueryBuilder()
+     let personTable = PersonTable()  // PersonTable() is a previously defined `Table` class
+     let description = try personTable.name.build(queryBuilder: queryBuilder)
+     print(description)
```

Comment:
My description prints - personTable.firstName I suggest to make this simpler you change the code at the top of the class to reference name instead of firstName, but you'll need to check the rest of the file for use of firstName elsewhere....
See line 32 `let name = Column("firstName", String.self, notNull: true)`